src/containers/status/Status.js

```diff
@@ -33,6 +33,7 @@ function Status ({ className }) {
   const watcherConnection = useSelector(selectConnection);
   const byzantineChain = useSelector(selectByzantine);
   const isSynced = useSelector(selectIsSynced);
```

Comment:
I think this should completely replace the isSynced logic. it's a bit confusing having both now. isSynced is still being used in the watcherStatus and to determine whether a user can deposit as well, so that should all be updated/replaced with this new check.